### PR TITLE
TorQ Cloud Post Merge Fixes

### PR DESCRIPTION
**Pull request by mattdeshAQ**
This PR contains fixes for merge conflicts/errors that have arisen after merging the main branches together in torq5_develop. (An FSP partner will also have a PR)

### CODE REVIEW

File: code/rdb/datastripe.q [hunk 1]
```diff
@@ -73,4 +73,9 @@ initdatastripe:{
 getaccess:{[] `location`table xkey update location:.proc.procname,proctype:.proc.proctype from .ds.access};
 
 // function to update the access table in the gateway. Takes the gateway handle as argument
-updategw:{[h]neg[h](`.ds.updateaccess;getaccess[]);};
+updategw:{[h]
+
+	newtab:getaccess[];
+ 	neg[h](`.ds.updateaccess;newtab);
+
+ 	};
```
Comment (jonathonmcmurray) [inline code/rdb/datastripe.q:81]: this seems like a regression? the updated version should work the same, is there a reason to revert it back to this version?

**Review by jonathonmcmurray — APPROVED** [review]
Mostly fine, one thing I'm not sure should be here

File: code/rdb/datastripe.q (hunk 1 above)
Comment (mattdeshAQ) [reply code/rdb/datastripe.q:81]: looks to be accidentally pushed during testing - I'll sort that one out now

**Review by CameronAquaQ — APPROVED** [review]
Looks good.

**Review by jonathonmcmurray — APPROVED** [review]

**Pull request merged**